### PR TITLE
Fix alert filter doesn't work properly

### PR DESCRIPTION
**Pull request by mactanxin**
ticket: #19828

### CODE REVIEW

**Review by pchaussalet — APPROVED** [review]
LGTM

File: ui/sections/system/inspectors/alert.reel/table-predicates.reel/table-predicates.js [hunk 1]
```diff
@@ -3,11 +3,31 @@ var Component = require("montage/ui/component").Component;
...
-                value: null
+            if(this.rows && this.rows.length == 0) {
+                return {
+                    property: "class",
+                    operator: null,
+                    value: null,
+                    _disabled: false
+                }
+            } else if(this.rows && this.rows.length == 1){
+                switch (this.rows[0].property) {
```
Comment (pchaussalet) [inline ui/sections/system/inspectors/alert.reel/table-predicates.reel/table-predicates.js]: You could have used something like
```
property = _.head(_.reject(["class", "severity"], function(x) { return x === self.rows[0].property; }));
```

**Review by pchaussalet — CHANGES_REQUESTED** [review]

**Comment by mactanxin** [comment]
updated

**Review by pchaussalet — APPROVED** [review]
LGTM

**Pull request merged**